theme/static/css/hydroshare_core.css

```diff
@@ -682,19 +712,44 @@ li.account p{
 	text-transform: none;
 	word-wrap: break-word;
 }
+
 #cogs-menu,
 #profile-menu {
     display: none;
     text-transform: initial;
 }
+
 #profile-menu > .dropdown-toggle {
-    font-size: 14px;
+    padding: 0;
+}
+
+#profile-menu .dropdown-footer {
+    position: absolute;
+    bottom: 20px;
+    border-top: 1px solid #CCC;
+    width: 100%;
+    margin: -20px;
+	padding: 10px;
+	background: #DDD;
+	-webkit-border-bottom-right-radius: 4px;
+	-webkit-border-bottom-left-radius: 4px;
+	-moz-border-radius-bottomright: 4px;
+	-moz-border-radius-bottomleft: 4px;
+	border-bottom-right-radius: 4px;
+	border-bottom-left-radius: 4px;
```

Comment:
This should be resolved now. See https://github.com/hydroshare/hydroshare/blob/686-home-page-revamp/theme/static/css/hydroshare_core.css#L724

Not sure why it looks over indented in the code block above.